doc/latex.rst

```diff
@@ -189,14 +189,25 @@ The available styling options
     default ``true``. Tells whether long lines in :rst:dir:`code-block`\ 's
     contents should wrap.
 
+``literalblockcappos``
```

Comment:
Indeed, this PR partially addresses #1723 but only for code-blocks. It is not complete feature.

thanks for review, now merging, I am online for a few days.